certified-connectors/HTML To PDF By Pascalcase/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#18186d",
+    "capabilities": [],
+    "publisher": "Vikrant Upadhyay"

```

Comment:
@VikrantUpadhyay1 Please add the "stackOwner" property. Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json). 
```
"stackOwner": "Name of the owner of the service or product."
```